tensorflow/training/docker/2.5/py3/cu112/Dockerfile.gpu

```diff
@@ -211,26 +211,29 @@ RUN ln -s $(which ${PYTHON}) /usr/local/bin/python \
 # install PyYAML==5.1.2 to avoid conflict with latest awscli
 # # python-dateutil==2.8.0 to satisfy botocore associated with latest awscli
 RUN ${PIP} install --no-cache-dir -U \
+   # Protobuf above this version appears to result in errors in horovod.
```

Comment:
I believe we need TF 2.5.3 in this container

Reply:
Upgraded the patch version for training to 2.5.3